docs/configuration/recording_rules.md

```diff
@@ -82,6 +82,11 @@ name: <string>
 # rule can produce. 0 is no limit.
 [ limit: <int> | default = 0 ]
 
+# Labels to add or overwrite before storing the result for its rules.
```

Comment:
"Add or overwrite" - Hmmm, this actually makes me think that things are slightly confusing:

* Group labels do not override rule labels
* However, group labels *do* override labels from the *query results* in rules

So the outermost layer does not override the layer below it, but it does override the innermost one. That's a bit like the `MM/DD/YYYY` date format to me :P But it probably still makes sense.

Could I suggest a slightly different docstring here then:

```
# Labels to apply to the outputs of rules in this group.
# When a rule query result or rule definition contains conflicting label names,
# the precedence order (from lowest to highest) is:
# 
# * Labels originating from the rule's query result
# * Labels defined on a rule group
# * Labels defined on an individual rule
```